src/spotted/handlers/sban.py

```diff
@@ -15,19 +16,24 @@ async def sban_cmd(update: Update, context: CallbackContext):
         context: context passed by the handler
     """
     info = EventInfo.from_message(update, context)
+    sban_fail = False
     if context.args is None or len(context.args) == 0:  # if no args have been passed
         banned_users = "\n".join(f"{user.user_id} ({user.ban_date:%d/%m/%Y %H:%M})" for user in User.banned_users())
         banned_users = "Nessuno" if len(banned_users) == 0 else f"{banned_users}"
         text = f"[uso]: /sban <user_id1> [...user_id2]\nGli utenti attualmente bannati sono:\n{banned_users}"
         await info.bot.send_message(chat_id=info.chat_id, text=text)
         return
-
     for user_id in context.args:
-        # the sban was unsuccessful (maybe the user id was not found)
-        if not User(int(user_id)).sban():
-            break
-    else:
-        await info.bot.send_message(chat_id=info.chat_id, text="Sban effettuato")
-        return
-
-    await info.bot.send_message(chat_id=info.chat_id, text="Uno o più sban sono falliti")
+        if User(int(user_id)).sban():
+            try:
+                await info.bot.send_message(
+                    chat_id=user_id, text="Sei stato sbannato da Spotted DMI, puoi tornare a postare!"
+                )
+            except Forbidden:
+                pass
```

Comment:
i do not remember if we keep track of the log, but I guess so, you could print the error log in the admin group as it's already happening for other errors

Reply:
This has been handled with 8cbeda9